stories/Tooltip/Tooltip.story.jsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { AutoExample } from '@catho/quantum-storybook-ui';
+import {
+  AutoExample,
+  Tab,
+  Title,
+  StoryContainer,
+  SimpleHighlight,
+} from '@catho/quantum-storybook-ui';
+import { Row, Col } from '../../components/Grid';
 import Tooltip from '../../components/Tooltip';
+import examples from './examples/examples';
+
+const TabExample = (
+  <Tab title="Examples">
+    <StoryContainer>
+      <Title as="h2">Examples</Title>
+      {examples.map(example => (
+        <Row key={example.code}>
```

Comment:
Some how, its not working.
![image](https://user-images.githubusercontent.com/3389749/67102127-53d0e480-f199-11e9-8b01-fef96d559325.png)